src/mono/mono/mini/interp/transform.c

```diff
@@ -4531,6 +4534,11 @@ generate_code (TransformData *td, MonoMethod *method, MonoMethodHeader *header,
 				(td->sp > td->stack && (td->sp [-1].type == STACK_TYPE_O || td->sp [-1].type == STACK_TYPE_VT)) ? (td->sp [-1].klass == NULL ? "?" : m_class_get_name (td->sp [-1].klass)) : "");
 		}
 
+		if (without_debug_info && in_offset == 0) {
+			interp_add_ins (td, MINT_SDB_INTR_LOC);
+			last_seq_point = interp_add_ins (td, MINT_SDB_SEQ_POINT);
+		}
```

Comment:
Couldn't we make this check idential to JIT behavior instead ? Modifying the checks below to : 
`if (td->gen_sdb_seq_points && ((!sym_seq_points && td->stack == td->stack_start) || (sym_seq_points && mono_bitset_test_fast (seq_point_locs, td->ip - header->code)))) {`

Reply:
I couldn't find td->stack_start, I tried this and worked:
`if (td->gen_sdb_seq_points && ((!sym_seq_points && td->stack == td->sp) || (sym_seq_points && mono_bitset_test_fast (seq_point_locs, td->ip - header->code)))) {`
Is it correct?